Widgets 2.0 (#4028)
DataDog/documentation: content/en/graphing/widgets/hostmap.md

---

> title: Hostmap Widget

Reviewer: ```suggestion
title: Host map Widget
```

kind: documentation
description: Display the Datadog hostmap in your dashboards.
further_reading:
- link: "graphing/dashboards/timeboard/"
  tag: "Documentation"
  text: "Timeboards"
- link: "graphing/dashboards/screenboard/"
  tag: "Documentation"
  text: "Screenboard"
---

The host map graphs any metric for any subset of hosts on the same host map visualization available from the main [Infrastructure Host Map][1] menu:

{{< img src="graphing/widgets/hostmap/hostmap.png" alt="Host Map" responsive="true" >}}

## Setup

{{< img src="graphing/widgets/hostmap/hostmap_setup.png" alt="Host Map Setup" responsive="true" >}}

### Configuration

> Configuration of the Hostmap widget works as the main [Hostmap page][1]:

Reviewer: ```suggestion
Configuration of the host map widget works as the main [host map page][1]:
```

> 1. Choose to display `host` or `containers`

Reviewer: ```suggestion
1. Choose to display `hosst` or `containers`
```

> 2. `Filter by` : Choose which hosts/containers to display

Reviewer: ```suggestion
2. `Filter by : Choose which hosts/containers to display
```

> 3. `Group by`: Aggregate your hosts/containers depending of one or several Tag.

Reviewer: ```suggestion
3. `Group by`: Aggregate your hosts/containers by one or several tags.
```

> 4. Choose a metric to fill your hostmap elements.

Reviewer: ```suggestion
4. Choose a metric by which to fill your host map elements.
```

> 5. Optional - Choose a metric to size your hostmap elements.

Reviewer: ```suggestion
5. Optional: choose a metric by which to size your host map elements.
```

> 6. Optional - Define a color palette with a `min` and `max` color palette value.

Reviewer: ```suggestion
6. Optional: define a color palette with a `min` and `max` color palette value.
```
